learnupon_exporter/management/commands/learnupon_export_enrollment_data.py

```diff
@@ -113,9 +113,9 @@ def export_enrollment_data(self, csv_file):
                 row["Enrollment Status"] = "completed"
                 row["Enrollment Started Date"] = first_block_viewed_at
```

Comment:
The client said that most records don't have a "Start Date", and "start date comes after the finish date".

Instead of using this value, could you use `enrollment.created date` instead?

Reply:
@pomegranited with this change `Enrollment Created Date` == `Enrollment Start Date` should I remove the `Enrollment Created Date` column?

Reply:
Nah keep it -- I think LearnUpon needs it for the uploads. It's ok if it's got the same value in.